scripts/SetMigrations.sql

```diff
@@ -44,7 +44,8 @@ VALUES
 	(1596289368633, 'V18CreateIgnoreLogs1596289368633'),
 	(1596299849081, 'V19MigrateReactionRoles1596299849081'),
 	(1597088981807, 'V20UpdateGuildEntityChecks1597088981807'),
-	(1597266996401, 'V21AddMissingChecks1597266996401');
-	(1598010877863, 'V22AddSelfStarCapabilities1598010877863')
+	(1597266996401, 'V21AddMissingChecks1597266996401'),
+	(1598010877863, 'V22AddSelfStarCapabilities1598010877863'),
+	(1598083459734, 'V23_AddGIData1598083459734')
```

Comment:
```suggestion
	(1598083459734, 'V23AddGIData1598083459734')
```

And rename the file and the class.

Reply:
The class is already named that, and file names don't follow VX_Something.